README.md

```diff
@@ -235,21 +239,18 @@ orchestrator.
 to send various types of traffic to each container under test.  For example the orchestrator is used to ping a container
 under test, and to be the ping target of a container under test.
 
-The [included example](test-network-function/generic_test_configuration.yml) defines a single container to be tested,
+The [included example](test-network-function/tnf_config.yml) defines a single container to be tested,
 and a single partner to do the testing.
 
-### cnf_test_configuration.yml - Operator and Container Test Configuration
-
-Testing operator and containers in specific roles is currently configured separately from the generic tests. This is
-configured using `cnf_test_configuration.yml`.
+### cnfs and operators
 
-`cnf_test_configuration.yml` defines the roles under which operators and containers are to be tested.
+The `cnfs` and `operators` sections define the roles under which operators and containers are to be tested.
 
-[The included example config](test-network-function/cnf_test_configuration.yml) is set up with some examples of this:
+[The included example config](test-network-function/tnf_config.yml) is set up with some examples of this:
 It will run the `"OPERATOR_STATUS"` tests (as defined in `testconfigure.yml`) against an etcd operator, and the
 `"PRIVILEGED_POD"` and `"PRIVILEGED_ROLE"` tests against an nginx container.
 
-A more extensive example is provided in [pkg/config/example.yaml](pkg/config/example.yaml)
+A more extensive example of these sections is provided in [pkg/config/example.yaml](pkg/config/cnf_example.yaml)
```

Comment:
Maybe we ought to move this into the `examples` directory at the root of the project.  That way it is dis-aggregated from regular source code.